beets/util/__init__.py

```diff
@@ -1094,7 +1102,7 @@ def asciify_path(path: str, sep_replace: str) -> str:
     return os.sep.join(path_components)
 
 
-def par_map(transform: Callable, items: Iterable):
+def par_map(transform: Callable[..., Any], items: Iterable[Any]) -> None:
```

Comment:
```suggestion
def par_map(transform: Callable[[T], None], items: Iterable[T]) -> None:
```
We can make this even more specific.